### DISCUSSION TITLE
Absorption coefficient G

### DISCUSSION THREAD

**Discussion by fil123** [Universite-Gustave-Eiffel/NoiseModelling#575] [Q&A] [answered]
Hi,
- G : the acoustic absorption of a ground (FLOAT between 0 : very hard and 1 : very soft), according to END scheme, infact "he END scheme [55], is similar to IS09613-2 [1] in that it allows for frequency-dependent ground effect over non-flat ground by defining equivalent heights (see Section 12.10) and using a dimensionless frequency-independent coefficient, ground factor, G, with values between 0 (acoustically hard) and 1 (acoustically soft) according to the type of ground [55]". I noticed importing OSM maps that G related to water basins have an assigned G equal to 1.  (https://www.openstreetmap.org/export#map=18/39.22652/9.20618).
- 
![image](https://user-images.githubusercontent.com/1425862/233982225-858e9433-6ea2-4e7d-bff8-54a8ace88167.png)

The color Blue is water, based on your classification G could be 0. I hope that in calculation you use (1-G). 
Can you clarify this point please? 
I hope hearing from you soon.
Best regards.
Filippo Locci

**Comment by nicolas-f** [comment]
Hi,

In the last release of NoiseModelling water surface are exported with G value of 0.3

https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/4.X/wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/Import_and_Export/Import_OSM.groovy#L928

@Nitnelav, why is it 0.3 instead of 0 ?

**Comment by Nitnelav** [comment, marked as answer]
Good question, I used the values from [Geoclimate](https://github.com/orbisgis/geoclimate/blob/5a3b52cb2a02d001e2d8f868611569107c9a64a7/geoindicators/src/main/resources/org/orbisgis/geoclimate/geoindicators/ground_acoustic_absorption.json) without questionning anything, it might be a mistake

**Comment by nicolas-f** [comment]
Ok, maybe it was arbitrary values on Geoclimate, we should use the values from Cnossos:

https://noisemodelling.readthedocs.io/en/latest/Input_ground.html#id1

**Comment by pierromond** [comment]
It should be 0 then... :)
